stdlib/public/RemoteInspection/TypeLowering.cpp

```diff
@@ -1984,6 +1985,8 @@ class EnumTypeInfoBuilder {
           // We don't have typeinfo; something is very broken.
           Invalid = true;
           return nullptr;
+        } else if (Case.Indirect) {
+          ++IndirectPayloadCases;
```

Comment:
Note that cases that are both indirect _and_ generic used to be counted as Generic; now they're counted as "Indirect".

Reply:
This didn't work, so I changed it to instead count indirect cases as "non-generic/non-empty".  That seems to be a better fit.